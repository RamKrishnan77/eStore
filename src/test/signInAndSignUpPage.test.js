import {shallow} from 'enzyme'
import React from 'react'
import SignInAndSignUpPage from '../components/sign-in-and-sign-up.component'

//Snapshot testing of SignInAndSignUpPage using Eznymeâ€™s Full DOM rendering 

it('expect to render SignInAndSignUpPage', () => {
    expect(shallow(<SignInAndSignUpPage></SignInAndSignUpPage>)).toMatchSnapshot()
})

