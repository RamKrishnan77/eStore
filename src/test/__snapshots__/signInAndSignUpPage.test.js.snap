// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render SignInAndSignUpPage 1`] = `
<div
  className="sign-in-and-sign-up"
>
  <withRouter(SignIn) />
  <SignUp />
</div>
`;
